pkg/operator/operator.go

```diff
@@ -93,11 +93,34 @@ type Config struct {
 	RebootWindowLength   string
 	Namespace            string
 	LockID               string
+	ResourceLock         string
```

Comment:
We don't pass a lock here, just it's type.
```suggestion
	LockType         string
```